QuickReference.ipynb

```diff
@@ -0,0 +1,402 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 0. Setup\n",
+    "Please make sure your environment is set up according to the instructions here: https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/00_SETUP.md\n",
```

Comment:
Suggest turning this into a proper link i.e. \[here](https://github.com/NASA-NAVO/aas_workshop_2020_winter/blob/master/00_SETUP.md).

Reply:
Good idea.  I'll do it for the known issues as well.

Do you think I should do it the astropy and pyvo doc links as well?

Reply:
I am on the fence. One the on hand, it looks cleaner. On the other hand it might be useful for people to see those urls. I could go either way.